README.md

```diff
@@ -2,28 +2,65 @@
 
 A JavaScript library for managing authentication for [Storyblok](https://www.storyblok.com) apps.
 
+## Migrating v1 to v2
+
+The `@storyblok/app-extension-auth` v1 stored the access token in a cookie. However, in a future version of v2, the library will offer an adapter pattern, allowing you to store the access token elsewhere, such as in a dedicated database. And here are the breaking changes:
```

Comment:
Not sure how to phrase this 🤔 what do you think?

Reply:
This is fine, maybe adding one more sentence to make it clearer, but its just nitpicking. 

```suggestion
The `@storyblok/app-extension-auth` v1 stored the access token in a cookie. However, in a future version of v2, the library will offer an adapter pattern, allowing you to store the access token elsewhere, such as in a dedicated database. The following changes abstract the code to make this transition easier in the near future:
```